status/discover.go

```diff
@@ -39,147 +34,6 @@ func getClient() (kubernetes.IstioClientInterface, error) {
 	return client, nil
 }
 
-func checkIfQueryBasePath(ns string, service string) (path string, err error) {
-	path = ""
-	client, err := getClient()
-	// Return if there is a problem with the client
-	if err != nil {
-		return path, err
-	}
-	// Get the service
-	svc, err := client.GetService(ns, service)
-	if err != nil {
-		return path, err
-	}
-	labelSelectorService := labels.Set(svc.Spec.Selector).String()
-	deployments, err := client.GetDeploymentsByLabel(ns, labelSelectorService)
-
-	if err != nil {
-		return path, nil
-	}
-
-	switch len(deployments) {
-	case 0:
-		log.Debugf("[TRACING] Kiali didn't found a deployment for service %s", service)
-	case 1:
-		if len(deployments[0].Spec.Template.Spec.Containers) > 0 {
-			for _, v := range deployments[0].Spec.Template.Spec.Containers[0].Env {
-				if v.Name == "QUERY_BASE_PATH" {
-					path = v.Value
-					break
-				}
-			}
-		}
-	default:
-		log.Debugf("[TRACING] Kiali found 2 or + deployments for service %s", service)
-	}
-
-	return path, nil
-}
-
-func getPathURL(endpoint string) (path string) {
-	u, err := url.Parse(endpoint)
-	if err != nil {
-		return ""
-	}
-	return u.Path
-}
-
-func discoverTracingService() (service string, port int32) {
```

Comment:
does this mean Kiali server no longer attempts to auto-discover jaeger?

Reply:
No, the user must set the configuration we dicussed about it

Reply:
There are so uses cases like proxys etc....so this task is not perfect so we decided to let the user configure it

Reply:
I think we may want to align with Prometheus here which is not auto-discovered.
Auto-Discovery has introduced more complexity and I think we may want to reduce the number of cornercases and have a more clean config process.